package.json

```diff
@@ -36,23 +35,29 @@
     "jsdoc-fresh": "^1.0.2",
     "linkinator": "^1.4.0",
     "mocha": "^6.1.4",
-    "prettier": "^1.17.0"
+    "gts": "^0.9.0",
+    "prettier": "^1.17.0",
+    "typescript": "~3.5.3"
   },
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
-    "fix": "eslint . --fix",
-    "lint": "eslint .",
-    "test": "c8 mocha test/*.js",
+    "lint": "gts check && eslint .",
```

Comment:
@alexander-fenster we should work to get the initially generated `package.json` more aligned with this one, there are a few differences:

* we still run eslint, for the benefit of the `samples` folder.
* we use `c8` for code coverage.
* we have doc generation and doc tests.

Reply:
Filed as https://github.com/googleapis/gapic-generator-typescript/issues/88